src/types/Conditional.js

```diff
@@ -11,7 +11,7 @@ export type AgeConditional = {
   condition_type: 'Age',
   operator: '==' | '!=' | "<" | "<=" | ">" | ">=",
   quantity: number,
-  unit: string
+  unit: 'years' | 'months' | 'weeks' | 'days' | 'hours' | 'minutes' | 'seconds'
```

Comment:
Similar deal here, if these are all identical (which it looks like they are) let's encapsulate that in a "TimeUnit" type so we can reference it in multiple places.